### PR TITLE
update README

### PR DESCRIPTION
**Pull request by LarryUllman**
* Updates the README to ~match edits made to https://www.flexpa.com/docs/getting-started/quickstart
* Some minor edits to the application content

### CODE REVIEW

**Comment by LarryUllman** [comment]
cc @healthbjk

File: README.md [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
 git clone https://github.com/flexpa/quickstart.git
-cd quickstart
 ```
 
-## Update environment variables
+## Set API keys
 
-You'll need to add your Flexpa publishable and secret keys to a `.env` file in the client and server directories.
-Rename the files `.env.template` to `.env` in both directories and update the values to your publishable and secret keys.
+Second, create an account in our [developer portal](https://portal.flexpa.com/) to get your test mode keys. You can [contact us](https://flexpa.typeform.com/intake) later for live mode access.
```
Comment (healthbjk) [inline README.md:19]: > You can [contact us](https://flexpa.typeform.com/intake) later for live mode access.

They actually can sign up in Portal and request Live Mode access there.

**Review by healthbjk — CHANGES_REQUESTED** [review]
Looks good, one small change

File: README.md (hunk 1 above)
Comment (LarryUllman) [reply README.md]: Thanks for catching. Just pushed that tweak!

**Review by healthbjk — APPROVED** [review]

**Pull request merged**